gemsiv/delaunay/quadedge.C

```diff
@@ -1,4 +1,9 @@
 #include "quadedge.h"
+#include "../../dummy.h"
```

Comment:
I renamed it to fakeirisgl to make it clear which API was used and documented it in fakeirisgl.h.